cloudify_rest_client/utils.py

```diff
@@ -60,3 +60,17 @@ def is_kerberos_env():
     if os.path.exists('/etc/krb5.conf') and find_executable('klist'):
         return True
     return False
+
+
+def get_folder_size_and_files(path):
```

Comment:
Can you add a unittest for this in case someone does something clever with it later?